test/functions.jl

```diff
@@ -13,45 +13,44 @@
 
     @objective(m, Max, dot(v,x))
 
-    @NLconstraint(m, sum(w[i]*x[i]^2 for i=1:5) <= 45)   
+    @NLconstraint(m, sum(w[i]*x[i]^2 for i=1:5) <= 45)
 
     MOIU.attach_optimizer(m)
-    bm = JuMP.backend(m)
-    options = bm.optimizer.model.optimizer.options
+    options = JuMP.unsafe_backend(m).options
```

Comment:
It looks to me this was the only breaking change that impacted Juniper in the move from MOI v0.10 to 1.0? Is that correct?

Reply:
It's a change in how JuMP manages its backend. 

I don't think we ever encouraged people to call `.optimizer.model.optimizer`, so this is arguably a bug-fix. We should have used `direct_model` if we cared about the backend. The new `unsafe_backend` provides a nice way to get the inner model for cases like this.